src/Tests/ExceptionReporter_Tests.cs

```diff
@@ -69,7 +71,46 @@ public void Can_Init_Config_Using_Object_Initializer()
 					FilesToAttach = new[] {"app.log"}
 				}
 			};
+
 			er.Show();
 		}
+
+		[Test]
+		[Ignore("UI")]
+		public static void TestRunManually()
```

Comment:
We might have to call this `ManualLocalizationTest` and might be best in it's own file eg `ExceptionReporter_ManualTests.cs`

Reply:
> We might have to call this `ManualLocalizationTest` and might be best in it's own file eg `ExceptionReporter_ManualTests.cs`

Yes, I agree with you.

Should I add two tests `ManualLocalizationRuTest` and `ManualLocalizationEnTest`?

Reply:
> We might have to call this ManualLocalizationTest and might be best in it's own file eg ExceptionReporter_ManualTests.cs

Done